docker/README.md

```diff
@@ -15,3 +21,20 @@ docker run -d -p 8771:8771 -e MIRA_NEO4J_URL=bolt://0.0.0.0:7687 mira_dkg:latest
 
 This exposes a REST API at `http://localhost:8771`. Note that the `-d` flag
 runs the container in the background.
+
+## MIRA Metaregistry
+
+You can build the metaregistry with:
+
+```shell
+docker build --file metaregistry.Dockerfile --tag mira_metaregistry:latest .
+```
+
+Once the build finished, you can run the container locally as:
+
+```shell
+docker run --detach -p 8772:8772 \
+    -e MIRA_NEO4J_URL=bolt://0.0.0.0:7687 \
+    -e MIRA_REGISTRY_CONFIG=/Users/cthoyt/dev/mira/mira/dkg/metaregistry/epi.json \
```

Comment:
@bgyori this doesn't work as expected, since giving a path here is meaningless inside the docker file. What do you think the best way to pass an external configuration into this pre-built image is?

Reply:
I thought the configuration would happen during build time, the config simply being copied into the image. It is still possible to do what you're doing above if an appropriate path is mounted to connect the image to the host, but I don't think that's necessary, we can just configure the registry during build time.